src/main/resources/hudson/model/ExternalJob/newJobDetail.jelly

```diff
@@ -24,5 +24,5 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <div>
-  ${%body}
+  ${it.description}
```

Comment:
That is precisely why this will be inappropriately escaped: XSS defense is activated against something which you the programmer know is safe.